pkgs/development/tools/sauce-connect/default.nix

```
@@ -4,18 +4,18 @@ with lib;
 
 stdenv.mkDerivation rec {
   name = "sauce-connect-${version}";
-  version = "4.5.1";
+  version = "4.5.4";
 
   src = fetchurl (
```

maybe bring in the hostPlatform namespace so you can use just "system" instead of that long path?
```suggestion
  src = with stdenv.hostPlatform; fetchurl (
```